addons/mail/static/src/models/message/message.js

```diff
@@ -245,6 +267,30 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * Performs DOM transformation needed to display the message.
+         * It uses a string representation of the DOM as we don't manipulate directly the real DOM with OWL. 
+         * 
+         * @param {String} content dom string rep on which to apply all the dom maniplation
+         * @returns processed content
+         */
+        processDomTransformation(content, isSafe = false) {
+            if (!content) return "";
+            if (!isSafe) {
+                content = _.escape(content);
+            }
```

Comment:
@mart-e All the data injected through the .innerHTML attribute go through this function. 
Would a simple escaping here like I did be enough ? It fixes the example your provided, but I'm not familiar with other ways it could get used.

Reply:
Yes, it fixes the problem here. As a general rule, you should avoid using `innerHtml` (use `textContent`) or constructing HTML by hand (use `createElement`, `setAttribute`, etc.) whenever possible.  Always be defensive and assume the content can not be trusted. Don't forget to include HTML in your test cases to make sure the escaping is correctly done once (and zero or twice).